canopen/objectdictionary/__init__.py

```diff
@@ -15,17 +15,22 @@
 logger = logging.getLogger(__name__)
 
 
-def export_od(od, dest: Union[str, TextIO, None] = None, doc_type: Optional[str] = None):
-    """ Export :class: ObjectDictionary to a file.
+def export_od(
+    od: ObjectDictionary,
+    dest: Union[str, TextIO, None] = None,
+    doc_type: Optional[str] = None
+) -> Union[str, None]:
+    """Export an object dictionary.
 
     :param od:
-        :class: ObjectDictionary object to be exported
+        The :class:`!ObjectDictionary` object to be exported.
```

Comment:
Perhaps it is better to spell it out; the param list will include the link to ObjectDictionary (and there is no need to link to it twice in one line.

```suggestion
        The object dictionary to be exported.
```

Reply:
IMO, it is better to spell it out; if the reviewer disagrees, one can simply revert the offending commit.